src/theory/arith/arith_rewriter.cpp

```diff
@@ -975,7 +1030,7 @@ RewriteResponse ArithRewriter::postRewriteTranscendental(TNode t)
             if (rem.isNull())
             {
               return RewriteResponse(REWRITE_DONE,
-                                     nm->mkConstReal(Rational(0)));
+                                     ensureReal(nm->mkConstReal(Rational(0))));
```

Comment:
why is this necessary?

Reply:
It's not, this was the byproduct of one the many things I was trying :)  I'll revert.